### PR TITLE
Storage Client Library 0.10.0 - GA preview

### PR DESCRIPTION
**Pull request by hasonmsft**
Storage Client Library 0.10.0 - GA preview

### CODE REVIEW

**Comment by hasonmsft** [comment]
The travis CI has some intermediate build issue. Close and re-open it to rebuild.

**Comment by hasonmsft** [comment]
The travis CI has some intermediate build issue. Close and re-open it to rebuild.

File: lib/common/util/sr.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ var SR = {
   INVALID_RANGE_FOR_MD5: 'The requested range should be less than 4MB when contentMD5 is expected from the server',
   INVALID_SAS_VERSION: 'SAS Version ? is invalid. Valid versions include: ?.',
   INVALID_SAS_TOKEN: 'The SAS token should not contain api-version.',
-  INVALID_SIGNED_IDENTIFIERS: 'Signed identifiers need to be an array.',
+  INVALID_SIGNED_IDENTIFIERS: 'Signed identifiers need to be an object with key as the id and the value as the access policy.',
```
Comment (yaxia) [inline lib/common/util/sr.js]: hash object

Comment (hasonmsft) [inline lib/common/util/sr.js]: Thanks. Updated.

**Pull request merged**